<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryModellingEditor</class>
 <widget class="QDialog" name="GeometryModellingEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1480</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometry Modelling</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBoxGeometrySyntax"/>
      </item>
      <item>
       <widget class="QFrame" name="frameSyntaxGenerator">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="NewBtn">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OpenBtn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SaveBtn">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SaveAsBtn">
           <property name="text">
            <string>Save as</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PrintBtn">
           <property name="text">
            <string>Print</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CopyBtn">
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CutBtn">
           <property name="text">
            <string>Cut</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PasteBtn">
           <property name="text">
            <string>Paste</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="UndoBtn">
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RedoBtn">
           <property name="text">
            <string>Redo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FontBtn">
           <property name="text">
            <string>Font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BoldBtn">
           <property name="text">
            <string>Bold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ItalicBtn">
           <property name="text">
            <string>Italic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="UnderlineBtn">
           <property name="text">
            <string>Underline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AboutBtn">
           <property name="text">
            <string>About</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ExitBtn">
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1366</width>
           <height>497</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
